test/e2e/basic/available.go

```diff
@@ -18,11 +18,16 @@ import (
 )
 
 var _ = ginkgo.Describe("[basic][available] Node Tuning Operator availability", func() {
+	const (
```

Comment:
This `const` declaration is repeated many times under the same pkg `test/e2e/basic`, 
Is good to have per-test values, so I agree with the duplication, just wanted to have a short discussion on this, but is not a blocker. the more verbosity on test code, the better.
worth spending some time discussing this.

Reply:
Yeah, thanks for bringing this up.  So I decided to have this per test rather than per-package because it gives one more flexibility to decide what you want specifically for each of the tests.  For reboots, I've found in the past that 5 minutes is not enough, so I switched to 20 minutes.  Hence the idea to keep this per-tests to easily adjust this in the future if needed.